backend/app/usecase/match/contains_any.go

```diff
@@ -0,0 +1,22 @@
+package match
+
+import "strings"
+
+var _ Keyword = (*ContainsAny)(nil)
+
+// ContainsAny checks if an input contains any element of a list.
+type ContainsAny struct {
+}
+
+// IsMatch checks if the input contains any word.
+func (c ContainsAny) IsMatch(words []string, input string) bool {
+	if len(words) == 0 {
+		return true
+	}
```

Comment:
```suggestion
```
In my previous comment, I meant the implementation was right. This should be false. And this is already handled by the logic below.
Does the `input` contain any words from the `words` list? No(when the `words` list is empty).